com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/TextureSetGenerator.java

```diff
@@ -417,11 +464,37 @@ public static TextureSetResult calculateLayout(List<Rect> images, List<SpriteGeo
         return calculateTextureSetResult(layout, imageHulls, useGeometries, iterator);
     }
 
+    // Public api
+    // Convert from image space coordinates to
+    public static TextureSetResult createTextureSet(List<TextureSetLayout.Layout> layouts, AnimIterator iterator) {
```

Comment:
This is called from the texture packer extension, with a pre-created list of layouts.